# Size is 1x2 tuple
uint16[] size
flatten2DArray[] parallelMap


